src/MediaFrameListenerBridge.cpp

```diff
@@ -396,6 +400,14 @@ void MediaFrameListenerBridge::onMediaFrame(DWORD ignored, const MediaFrame& fra
 
 				//TODO: move out of here
 				VideoLayerSelector::GetLayerIds(packet);
+				
+				// Set frame resolution if needed
+				if ((video->GetWidth() == 0 || video->GetHeight() == 0) && 
+				    (packet->GetWidth() != 0 && packet->GetHeight() != 0))
+				{
+					width = video->GetWidth();
```

Comment:
As we have removed setting the width/height on the video frames, we need to use the packet info instead
```suggestion
					width = packet->GetWidth();
```